javascript/ql/lib/semmle/javascript/ApiGraphs.qll

```diff
@@ -860,6 +865,15 @@ module API {
                   .getStaticMember(name, DataFlow::MemberKind::getter())
                   .getAReturn()
           )
+          or
+          exists(Function fun, DataFlow::InvokeNode invoke, int argIndex, Parameter rest |
```

Comment:
So this is modelling the parameters at (or after) the rest parameter, and modelling each entry as an array store into the spread argument.  
And this is then only used for the `getParameter` method to be able to retrieve the correct parameter when there are spread arguments?

Reply:
Yes. Added a comment to clarify.

Reply:
> And this is then only used for the getParameter method to be able to retrieve the correct parameter when there are spread arguments?

Ah, actually no, it's more than that. If a library model expects an argument to contain array elements, this will allow the model to find those array elements when the array was created by passing arguments into a rest parameter.